pages/index.js

```diff
@@ -1,19 +1,12 @@
 import { useEffect } from 'react'
-import { withRouter } from 'next/router'
+import { useRouter } from 'next/router'
 
-import Route from './_app/route'
-
-import { withGlobalStore } from 'store'
-
-const Index = ({ router, store }) => {
+// Unfortunately, our real homepage is missing
+const Home = () => {
+  const router = useRouter()
   useEffect(() => {
-    const route = new Route({
-      dataProvider: store.dataProvider.id,
-      activity: store.activity.path,
-    })
-    router.push(route.href, route.as)
+    router.push('/data-providers')
   })
-  return null
 }
```

Comment:
you should probably leave `return null` here. Since router.push is async it doesn't mean that Home component gets redirected before render. It may happen after render.

Reply:
I know. I hate that React cannot render `undefinded` and makes me to explicitly return `null`

Reply:
me too :(